packages/semi-ui/carousel/CarouselArrow.tsx

```diff
@@ -50,6 +51,7 @@ class CarouselArrow extends React.PureComponent<CarouselArrowProps> {
                     className={rightClassNames} 
                     onClick={next}
                     {...get(this.props, 'arrowProps.rightArrow.props')}
+                    x-semi-prop="arrowProps.rightArrow.children"
```

Comment:
这个地方是不是漏了一级，上面是 ```get(this.props, 'arrowProps.rightArrow.props')```，比54行多了 props
```suggestion
                    x-semi-prop="arrowProps.rightArrow.props.children
```

Reply:
这里没有问题。这里是为了标识 children。

![image](https://user-images.githubusercontent.com/26477537/167853967-44f704b0-36a7-4c13-99eb-ae15c8e5f956.png)